internal/cmd/db.go

```diff
@@ -61,18 +81,28 @@ func getDatabase(client *turso.Client, name string) (turso.Database, error) {
 	return turso.Database{}, fmt.Errorf("database %s not found. List known databases using %s", internal.Emph(name), internal.Emph("turso db list"))
 }
 
-func getDatabaseNames(client *turso.Client) []string {
+func getDatabasesCacheOrAPI(client *turso.Client) ([]turso.Database, error) {
```

Comment:
⛏️: nit about naming. `getDatabases` is just fine. Callers shouldn't be worried about where the data is coming from, they should just have an way to get the value.